internal/deviations/deviations.go

```diff
@@ -452,6 +452,10 @@ var (
 
 	p4rtUnsetElectionIDUnsupported = flag.Bool("deviation_p4rt_unsetelectionid_unsupported", false, "Device does not support unset Election ID")
 
+	P4rtUnsetElectionIDPrimaryAllowed = flag.Bool("deviation_p4rt_unsetelectionid_primary_allowed", false, "Device allows unset Election ID to be primary")
```

Comment:
Please use the new style guide to  add deviations i.e by using accessor methods and making this parameter private (lowercase)

Reply:
Refer to https://github.com/openconfig/featureprofiles/pull/1397/files for example